path_oidc.go

```diff
@@ -454,6 +454,13 @@ func (b *jwtAuthBackend) createOIDCRequest(config *jwtConfig, role *jwtRole, rol
 
 	if config.hasType(responseTypeIDToken) {
 		options = append(options, oidc.WithImplicitFlow())
+	} else {
```

Comment:
Should we have a strict type for PKCE response type? The else here seems too broad, especially if we add more flows in the future?

Reply:
Makes sense to test for `responseTypeCode` since PKCE only (?) applied there.

Reply:
The [RFC](https://datatracker.ietf.org/doc/html/rfc7636) says that "clients utilizing the Authorization Code Grant are susceptible to the authorization code interception attack". So I agree we should just check for `responseTypeCode`. I will update the PR. Thanks!